modules/packages/ZMQ.chpl

```diff
@@ -884,9 +884,16 @@ module ZMQ {
         // conditionally have ZeroMQ free the memory.
         //
         // Note: the string factory below can throw DecodeError
-        var copy = if isString(T) then createStringWithNewBuffer(x=data)
-                   else parallelCreateBytesWithNewBuffer(data.localize().buff,
+        // var copy: string;
+        var copy;
+        if isString(T) {
+          var tmp:string = data;
+          copy = data;
```

Comment:
This results in one extra copy, I think.

Could this be just:

```
var copy = if isString(T) then data else parallelCreateBytes...
``` 

i.e., replace the deprecated factory with just the string.